hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/protocol/datatransfer/PacketReceiver.java

```diff
@@ -80,8 +80,8 @@ public class PacketReceiver implements Closeable {
   static {
     Configuration conf = new HdfsConfiguration();
     MAX_PACKET_SIZE = conf.getInt(HdfsClientConfigKeys.
-                    DFS_DATA_TRANSFER_MAX_PACKET_SIZE,
-            HdfsClientConfigKeys.DFS_DATA_TRANSFER_MAX_PACKET_SIZE_DEFAULT);
+        DFS_DATA_TRANSFER_MAX_PACKET_SIZE,
+        HdfsClientConfigKeys.DFS_DATA_TRANSFER_MAX_PACKET_SIZE_DEFAULT);
```

Comment:
BTW, please revert this irrelevant style change in order to help easy review and backport, @cxzl25 .